### PR TITLE
changes/fixes for singulo

### PR DESCRIPTION
**Pull request by Ilya246**
## About the PR

changes:
- singularities are now always InAir as opposed to being OnGround if level 1 and InAir otherwise
- rescale level 5 singularity (was too large previously)
- add MovementIgnoreInAirComponent
- add sentient singularity prototype for admemes or future PRs

**Media**

singularity ghost role
image

rescaled level 5 singularity
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

changes not player-facing, no CL

### CODE REVIEW

File: Content.Shared/Movement/Systems/SharedMoverController.cs [hunk 1]
```diff
@@ -147,12 +147,22 @@ public override void UpdateAfterSolve(bool prediction, float frameTime)
...
+            if (physicsComponent.BodyStatus != BodyStatus.OnGround)
+            {
+                var ev = new CanInAirMoveEvent(uid);
+                RaiseLocalEvent(uid, ref ev, true);
+                if (!ev.CanMove)
+                {
+                    UsedMobMovement[uid] = false;
+                    return;
+                }
+            }
```
Comment (metalgearsloth) [inline Content.Shared/Movement/Systems/SharedMoverController.cs:165]: This should be using CanMove as that gets cached as well.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Shared/Movement/Systems/SharedMoverController.cs (hunk 1 above)
Comment (Ilya246) [reply Content.Shared/Movement/Systems/SharedMoverController.cs:165]: a bit confused, which CanMove are you referring to here?

Comment (metalgearsloth) [reply Content.Shared/Movement/Systems/SharedMoverController.cs:165]: Raising an event every tick for movement is a really bad idea.

Comment (metalgearsloth) [reply Content.Shared/Movement/Systems/SharedMoverController.cs:165]: Also Mirror ended up doing this for her pr as I didn't see this comment so might be better to just use hers.

**Comment by metalgearsloth** [comment]
Re-open if you come back to it.

**Pull request closed without merging**